content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -445,7 +445,11 @@ systemctl restart containerd
 
 ### systemd
 
-To use the `systemd` cgroup driver, set `plugins.cri.systemd_cgroup = true` in `/etc/containerd/config.toml`.
+To use the `systemd` cgroup driver, go to `/etc/containerd/config.toml` and set
+
+[plugins.cri]`
+
+`systemd_cgroup = true`
 When using kubeadm, manually configure the
```

Comment:
Here is how it looks in preview:
![Screen Shot 2020-08-20 at 10 36 58](https://user-images.githubusercontent.com/13677519/90730984-b7540300-e2d1-11ea-8d78-b0a9d9e1050a.png)

It should be reformatted a bit to be more readable, I believe.

Reply:
To be honest, @RA489, I prefer the original text.

Reply:
I agree. I'd prefer to close this pull request.

Reply:
@bart0sh I have updated. PTAL.

Reply:
@RA489 thanks for the update. I like it better than current version as it shows which section of the configuration file to update.